### DISCUSSION TITLE
Did Certificate Renewal Patch Work?

### DISCUSSION THREAD

**Discussion by Stratlin** [cat-box/aws-foundry-ssl#17] [Q&A]
I recently received a letter from Let'sEncrypt informing me that my certificate was about to expire. Upon coming here I saw and followed the steps for the Certificate Renewal Patch. I still have some questions, however:

1. Do I need to reboot to make the patch take effect?
2. Is there graceful retry logic and if so, how frequently does it retry to renew?
3. Is there a way that I can confirm that my certificate has, indeed, been updated?

Thank you so much for your help and for making this great resource available to the community.

**Comment by cat-box** [comment]
sorry about the late reply, completely missed the notification for this

1. I would restart the server.
2. The cert will check if expiration is within the next 30 days on each reboot and every day at 12pm based on the server's clock
3. go to https://crt.sh and type in your domain. You should see a new entry with the date of renewal. Alternatively, you can SSH into the server and run the commands `sudo su` followed by `certbot certificates`. It should list the expiration date.
